src/main/java/hudson/plugins/audit_trail/AuditTrailPlugin.java

```diff
@@ -108,11 +108,15 @@
     public boolean getLogBuildCause() {
         return shouldLogBuildCause();
     }
-    
+
     public boolean shouldLogBuildCause() {
         return logBuildCause;
     }
 
+    public boolean getLogCredentialsUsage() { return shouldLogCredentialsUsage(); }
+
+    public boolean shouldLogCredentialsUsage() { return logCredentialsUsage; }
```

Comment:
Apparently getLogCredentialsUsage() is needed for Jelly. I was first wondering if some changes to Jelly were made so that shouldLogBuildCause() was also resolvable by Jelly, because getLogBuildCause() was deprecated, but it doesn't seem like it.
I could remove shouldLogCredentialsUsage() in favor of getLogCredentialsUsage(), but then we should also remove shouldLogBuildCause() and re-introduce getLogBuildCause() I guess.

`<f:checkbox name="logBuildCause" checked="${descriptor.logBuildCause}"/>` looks up the method getLogBuildCause(), I think.